src/test/java/LoginTests.java

```diff
@@ -47,28 +47,32 @@ public void loginValidEmailPassword(){
     }
 
     //Negative Test
-    //@Test
+    @Test
     public void loginWithInvalidEmailValidPassword() throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+
 
         //navigateToPage();
         String expectedUrl = "https://qa.koel.app/";
-        enterEmail("alekseikoksharov@testproio");
-        enterPassword("ak1234!@#$");
-        submit();
+//        enterEmail("alekseikoksharov@testproio");
+//        enterPassword("ak1234!@#$");
+//        submit();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.